frontend/src/components/workplan/TestSelectForm.js

```diff
@@ -20,7 +22,18 @@ function TestSelectForm(props) {
 
   useEffect(() => {
     mounted.current = true;
-    getFromOpenElisServer("/rest/test-list", getTests);
+    let testId = new URLSearchParams(window.location.search).get(
+      "test_id"
```

Comment:
rename to `testId`  to be consisted with old UI

Reply:
Thanks, updated to testId